api/controllers/userController.js

```diff
@@ -1,71 +1,58 @@
-const User = require('../models/user.js');
-const uuid = require('uuid');
+const { models } = require('../db');
 
-exports.create = (req, res) => {
-    if (!req.body) {
-        return res.status(400).send({
-            message: 'User content can not be empty!'
-        });
+async function create(req, res) {
+    try {
+        await models.user.create(req.body);
+        res.status(201).end();
+    } catch (error) {
+        console.log(error);
```

Comment:
Would be better to use the following signature of controller:
async function create(req, res. next) {
try {
//do smth
} catch (e) {
//handle error or return error to pass error to express
next(e) 
}

Reply:
recommendation was applied, 'next' was added to controller functions